octopusdeploy/config.go

```diff
@@ -23,7 +23,7 @@ func (c *Config) Client() (*client.Client, diag.Diagnostics) {
 	}
 
 	// This is intentional on the feature branch - Todo: remove when merging to main branch
-	proxyStr := "http://127.0.0.1:8866"
+	proxyStr := "http://172.21.224.1:8866"
```

Comment:
Please drop this change and keep it local for now.

Reply:
Nice catch, I forgot!